### DISCUSSION TITLE
When does apply COMPLETED.consume_material and WIP.consume_capacity as false?

### DISCUSSION THREAD

**Discussion by sgfisme** [frePPLe/frepple#436] [Q&A]
Just wondering to know what scenarios can apply COMPLETED.consume_material and WIP.consume_capacity as false.

**Comment by jdetaeye** [comment]
These settings are important when integrating with an ERP system.  And it can be quite tricky to understand and configure correctly.

Depending on the configuration of the ERP the consumed materials are booked from the inventory:
- at the start (eg picking components from the warehouse before production starts),  
- throughout the whole production order (eg consumption from a tank) 
- or at the end of the operation (eg "backflush" booking of consumed material when the operation is done).
